stackslib/src/chainstate/nakamoto/coordinator/tests.rs

```diff
@@ -283,8 +283,9 @@ fn test_simple_nakamoto_coordinator_bootup() {
 #[test]
 fn test_simple_nakamoto_coordinator_1_tenure_10_blocks() {
     let mut test_signers = TestSigners::default();
-    let mut peer = boot_nakamoto(function_name!(), vec![], test_signers.aggregate_public_key);
-    let private_key = peer.config.private_key.clone();
+    //let mut peer = boot_nakamoto(function_name!(), vec![], test_signers.aggregate_public_key);
+    //let private_key = peer.config.private_key.clone();
+    let private_key = StacksPrivateKey::new();
```

Comment:
Why is this a new key?  Can you continue to please use the same key so we get consistent output?